.npmignore

```diff
@@ -0,0 +1,3 @@
+test
```

Comment:
The only merit I can think of with `.npmignore` over the files prop in package.json is that you don't have to worry about adding a new entry in the files prop when you restructure or add a file.